galaxy_ng/app/api/v3/serializers/sync.py

```diff
@@ -0,0 +1,45 @@
+from galaxy_ng.app.constants import VALID_REMOTE_NAMES
+from pulp_ansible.app.models import CollectionRemote
+from rest_framework import serializers
+
+
+class SyncConfigSerializer(serializers.ModelSerializer):
+    created_at = serializers.DateTimeField(source='pulp_created')
+    updated_at = serializers.DateTimeField(source='pulp_last_updated')
+
+    class Meta:
+        model = CollectionRemote
+        fields = (
+            'name',
+            'url',
+            'auth_url',
+            'policy',
+            'requirements_file',
+            'created_at',
+            'updated_at',
+        )
+
+    def validate(self, data):
+        errors = {}
+        name = data['name']
+        requirements_file = data['requirements_file']
+
+        if name not in VALID_REMOTE_NAMES:
+            errors['name'] = (
+                f'{name} is not a valid remote name, possible values are {VALID_REMOTE_NAMES}'
+            )
+
+        if name == 'rh-certified' and requirements_file is not None:
```

Comment:
I don't think we need to enforce no requirements file for red hat certified. We won't offer it as an option in the UI, but if someone wants to back door the API and put one in, we don't have to stop them

Reply:
OK, I will change that. 

This is what I read on doc: 
```
requirements_file (string) – Explicit list of requirements to synchronize (Must be null for rh-certified repository and non-empty for community repository).
```